src/test/kotlin/org/rust/ide/annotator/AnnotationTestFixtureBase.kt

```diff
@@ -108,6 +108,19 @@ abstract class AnnotationTestFixtureBase(
         checkAfter = this::checkByText,
         testmark = testmark)
 
+    fun checkFixPartial(
```

Comment:
Is there a nicer way to do what I need for RsAnnotationTestBase#checkFixByTextWithLiveTemplate?